test/e2e/test_translate_proxy.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+###############################################################################
+# Integration testing for the MIE service proxy API for Amazon Translate
+#
+# PRECONDITIONS:
+# MIE base stack must be deployed in your AWS account
+#
+# Boto3 will raise a deprecation warning (known issue). It's safe to ignore.
+#
+# USAGE:
+#   cd tests/
+#   pytest -s -W ignore::DeprecationWarning -p no:cacheprovider
+#
+###############################################################################
+
+import urllib3
+import time
+import json
+
+
+def test_parallel_data(workflow_api, testing_env_variables, parallel_data):
+    workflow_api = workflow_api()
+
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
You might check with @ianwow on removing the insecure request warning. I know he did some work to address that (due to security concerns), but I don't recall the specifics.

Reply:
Removed this line of code.  Tested without it and didn't see any issues.